projects/public-search/src/app/patron-profile/patron-profile-loans/patron-profile-loan/patron-profile-loan.component.html

```diff
@@ -49,7 +49,7 @@
     <p-button
       size="small"
       class="ui:pointer-events-auto"
-      [pTooltip]="canExtend.reasons | arrayTranslate | join:' / '"
+      [pTooltip]="reasons | arrayTranslate | join:' / '"
```

Comment:
You can use `reasons.values() |...`  to avoid a getter which is not nice for template rendering. This will be refactor using signals and computed.